packages/cmcd-validator-library/src/keyValueValidator/keysValueValidator.js

```diff
@@ -14,6 +16,16 @@ const keyValValidator = (cmcdJson, errors, warnings, config, warningFlag = true)
     checkSidIsPresent(cmcdJson, warnings);
   }
 
+  // Check if all the specifickey have been received in the request
+  if (config?.specificKey) {
+    const keysReceved = Object.keys(cmcdJson);
+    config.specificKey.forEach((key) => {
+      if (!keysReceved.includes(key)) {
+        warnings.push(createWarning(warningTypes.specificKeysNotSended, key));
```

Comment:
Typo here, it should be 'sent'

Reply:
thank you! I'm going to change it right now